### PR TITLE
Update Mantid basic course

### LINKED ISSUE #35032: Update Mantid Basic Course
The following updates to the [Mantid Basic Course](http://docs.mantidproject.org/tutorials/mantid_basic_course/index.html#mantid-basic-course) were found to be needed during manual testing under issue #34955:

- [x] [04_displaying_2D_data](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/loading_and_displaying_data/04_displaying_2D_data.html#displaying-2d-data) - does not have the correct image or any information relating to the cutting tool added in PR #33765. _After reviewing the information provided on the page, it seems sufficient just to update the screenshots._

- [x] [06_formatting_plots.html](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/loading_and_displaying_data/06_formatting_plots.html#formatting-plots) - plot toolbar graphic needs updating, it does not include the new hide button introduced in PR #34614.

- [x] [1DPlotsHelp.html#basic-1d-plots](https://docs.mantidproject.org/nightly/plotting/1DPlotsHelp.html#basic-1d-plots) - all the script examples on this page are out of date (running them would give error "Legend has no attribute draggable"). It would be good to generate up to date examples from Mantid.

- [x] [03_workspaces.html#correct-binning](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/algorithms_workspaces_and_histories/03_workspaces.html#correct-binning) - plot generated from following the example gives [this colourfill](https://github.com/mantidproject/mantid/issues/34955#issuecomment-1385553972). Either need to update the screenshot in the course or correct the steps to allow users to produce a matching result.

- [x] [03_investigating_data.html#the-toolbar](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/connecting_data_to_instruments/03_investigating_data.html#the-toolbar) - buttons on this page need updating.

- [x] [03_investigating_data.html#selecting-arbitrary-sets-of-detectors](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/connecting_data_to_instruments/03_investigating_data.html#selecting-arbitrary-sets-of-detectors) - button positions have changed so the reference to positions here could do with being updated.

- [x] [03_investigating_data.html#picking-a-single-detector](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/connecting_data_to_instruments/03_investigating_data.html#picking-a-single-detector) - following the steps exactly as written causes the plotted curve to disappear as soon as the mouse moves off the detector. We should make it clear that the necessary steps are to click the `Add a single crystal peak` button and then click a detector in the instrument view (not just mouse over one) before moving to zoom in to the mini-plot.

- [x] [03_investigating_data.html#picking-a-tube](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/connecting_data_to_instruments/03_investigating_data.html#picking-a-tube) - similar to the above, except the necessary steps seem to be to click the tube selection button, click on a detector and then keep holding the mouse button down until the cursor is over the mini-plot. That way the plot's right click menu can then be used without the curve disappearing.

- [x] ~[04_masking_and_grouping.html#region-of-interest](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/connecting_data_to_instruments/04_masking_and_grouping.html#region-of-interest) - the last paragraph of this section looks like it should actually be under the Masking section above.~ On a second look this actually does seem to make sense, so it's simply a small typo that needs correcting here.

- [x] [06_sample_logs.html#basic-inspection](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/connecting_data_to_instruments/06_sample_logs.html#basic-inspection) - screenshot is of `BSS_11841_event` rather than `CNCS_7860_event`.

- [x] [06_sample_logs.html#algorithms](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/connecting_data_to_instruments/06_sample_logs.html#algorithms) - the paragraph underneath the screenshot in this section that begins:
> Look at the effect this filter has on the Temperature Sample Log and...

should say SampleTemp rather then Temperature Sample.

- [ ] [06_sample_logs.html#algorithms](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/connecting_data_to_instruments/06_sample_logs.html#algorithms) - underneath the paragraph described above, the explanation for the 0 relative time start is confusing and the plot that should appear under this paragraph does not appear, it just incorrectly displays `(Source Code)`. _This one should be handled as a separate issue as building locally renders the plots, so perhaps there's an issue with the Jenkins build - see #35214._

- [x] [07_invalid_sample_logs.html](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/connecting_data_to_instruments/07_invalid_sample_logs.html) - the dataset `ENGINX00228061_log_alarm_data.nxs` referenced on this page should be added to the TrainingCourseData.

- [x] [02_complex_case.html#functions-properties-setting-up-a-fit-model](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/fitting_data/02_complex_case.html#functions-properties-setting-up-a-fit-model) - the following paragraph is confusing as it mentions dragging on the plot after having set a FlatBackground, but this info should really be for the peak lines:
> Note that two fit functions have appeared in the Fit Property Browser called f0-Gaussian and f1-FlatBackground. Click on the triangles beside these functions to reveal their fit parameters. Just as with the StartX/EndX values, dragging on the plot and changing these values update each other.

- [x] [02_complex_case.html#tying-and-constraining-fit-parameters](https://docs.mantidproject.org/nightly/tutorials/mantid_basic_course/fitting_data/02_complex_case.html#tying-and-constraining-fit-parameters) - the `Tie > to Function` option works fine in the case of 2 peaks rather than 1 peak plus 1 background. The docs should be changed to make it clearer that the user should use `Tie > Custom Tie` for the examples it gives.

### PR DESCRIPTION
**Pull request by rbauststfc**
**Description of work.**

This PR makes the updates described on issue #35032 that were identified during manual testing for the 6.6 release. Note that point two (toolbar image updates) on the list of updates has been done in separate PR #35226.

**To test:**

1) Check that the text changes read OK.
1) Build the docs-html target to check that the documentation builds and that the new screenshots render OK.

Fixes #35032. 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/plotting/1DPlotsHelp.rst [hunk 1]
```diff
@@ -88,22 +88,22 @@ Click the generate a script button |GenerateAScript.png| on a 1D Plot:
...
+   axes.plot(MAR11060, color='#2ca02c', label='MAR11060: spec 3', wkspIndex=2)
+   axes.tick_params(axis='x', which='major', **{'gridOn': False, 'tick1On': True, 'tick2On': False, 'label1On': True, 'label2On': False, 'size': 6, 'tickdir': 'out', 'width': 1})
+   axes.tick_params(axis='y', which='major', **{'gridOn': False, 'tick1On': True, 'tick2On': False, 'label1On': True, 'label2On': False, 'size': 6, 'tickdir': 'out', 'width': 1})
    axes.set_title('MAR11060')
-   axes.set_xlabel('Time-of-flight ($\mu s$)')
-   axes.set_ylabel('Counts ($\mu s$)$^{-1}$')
-   axes.legend().draggable()
+   axes.set_xlabel('Time-of-flight ($\\mu s$)')
+   axes.set_ylabel('Counts ($\\mu s$)$^{-1}$')
+   legend = axes.legend(fontsize=8.0).set_draggable(True).legend
```
Comment (jhaigh0) [inline docs/source/plotting/1DPlotsHelp.rst:106]: Just wondering the reason for creating a `legend` variable if it isn't used in these examples. Is it to show that `axes.legend()` will return the legend it creates?

**Review by jhaigh0 — COMMENTED** [review]
Good updates to the training docs and their images, just left one questions about a code snippet, other than that all great.

File: docs/source/plotting/1DPlotsHelp.rst (hunk 1 above)
Comment (rbauststfc) [reply docs/source/plotting/1DPlotsHelp.rst:106]: Good question - these script examples are supposed to be demonstrating what you get when you click the "generate script" button on a plot. The script that's generated when you do that seems to include the legend variable. I'm not sure why that is, but that's why it gets included in the example in the documentation.

Comment (jhaigh0) [reply docs/source/plotting/1DPlotsHelp.rst:106]: Okay, I think it's fine for it to be there, just curious.

**Review by jhaigh0 — APPROVED** [review]

**Comment by rbauststfc** [comment]
@mantidproject/gatekeepers, if possible, it would be great if we could get this one in for the Mantid training we're delivering at ISIS on the 1st of March.

**Pull request merged**